homeassistant/components/notify/lametric.py

```diff
@@ -93,6 +93,9 @@ def send_message(self, message="", **kwargs):
             devices = lmn.get_devices()
         for dev in devices:
             if targets is None or dev["name"] in targets:
-                lmn.set_device(dev)
-                lmn.send_notification(model, lifetime=self._lifetime)
-                _LOGGER.debug("Sent notification to LaMetric %s", dev["name"])
+                try:
+                    lmn.set_device(dev)
+                    lmn.send_notification(model, lifetime=self._lifetime)
+                    _LOGGER.debug("Sent notification to LaMetric %s", dev["name"])
+                except Exception as e:
+                    _LOGGER.warning("Problem sending notification to LaMetric %s: %s", dev["name"], e)
```

Comment:
May I recommend upping this to something more reasonable like 120 characters? :)

Reply:
There is more information on this in the Style Guidelines. TLDR: This won't be changed

https://home-assistant.io/developers/development_guidelines/

Reply:
PEP8 is fine, I run CI on a large Python project and enforcing style guidelines is important. We've found the 80 character limit to be too constraining and have upped it a bit (especially since nobody develops on a flxed-width terminal console anymore). But whatevs. :)